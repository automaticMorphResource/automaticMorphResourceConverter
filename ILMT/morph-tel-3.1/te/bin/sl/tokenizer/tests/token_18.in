దాంతో 1981లో ఈ ప్రాంతాన్ని "వరల్డ్ హెరిటేజ్" ప్రకటించారు. 
