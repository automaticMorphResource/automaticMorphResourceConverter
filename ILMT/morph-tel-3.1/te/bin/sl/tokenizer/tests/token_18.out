<Sentence id="1">
1	దాంతో	unk
2	1981లో	unk
3	ఈ	unk
4	ప్రాంతాన్ని	unk
5	"	unk
6	వరల్డ్	unk
7	హెరిటేజ్	unk
8	"	unk
9	ప్రకటించారు	unk
10	.	unk
</Sentence>

